docker/chemml/src/run.py

```diff
@@ -44,7 +16,10 @@ def run_calculation(geometry_file, output_file):
         "density": pred[2]
     }
 
-    cjson.setdefault('properties', {}).update(properties)
+    # We will only output the properties
+    cjson = {
```

Comment:
I would repeat the smiles in the output. Once we support lists of smiles I think something of the form
`
[ { 'smiles': smiles, 'properties': properties } ]
`
Would work and the output ingest can then unroll the outputs.